jest.config.js

```diff
@@ -2,7 +2,7 @@ const config = require('@polkadot/dev/config/jest');
 
 module.exports = Object.assign({}, config, {
   moduleNameMapper: {
-    '@polkadot/api-(codec|format|jsonrpc|provider|rx)(.*)$': '<rootDir>/packages/api-$1/src/$2',
+    '@polkadot/api-(codec|jsonrpc|provider|rx)(.*)$': '<rootDir>/packages/api-$1/src/$2',
```

Comment:
API format is now not used in API anymore, so be like all the cool kids and remove a lot of line of code...